samples/meetings-attendance-report/csharp/assets/sample.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meetings-transcription-csharp",
```

Comment:
Why do we need this fine ?
If we need it, please change the name which contains "Transcription" ?

Reply:
We need this, as per Wajeed. Updated to this sample